test/spec/SearchResults/getFacetValues.js

```diff
@@ -407,6 +407,94 @@ test('getFacetValues(hierarchical) returns escaped facet values', function () {
   expect(facetValues).toEqual(expected);
 });
 
+test('getFacetValues(hierarchical) ignores `rootPath`', function () {
```

Comment:
I might be wrong, but the title feels misleading. `getFacetValues('type')` seems on the contrary to take into account `rootPath`, as the expected data only returns facet values within `cats`.

Reply:
good point, probably rootPath should only be taken in account for isRefined, not for anything else